src/containers/RunStep.tsx

```diff
@@ -63,6 +63,19 @@ const connector = connect(
         return
       }
 
+      if (customMode) {
+        if (variables.some((item: any) => item.customCenter === null)) {
+          dispatch(
+            setError(
+              t`Configuration error`,
+              t`Cannot calculate scores: Custom center is enabled and one or more of your variables has no custom 
+              center.`,
+            ),
+          )
+          return
+        }
+      }
```

Comment:
This isn't necessary since the "Run Tool" button is disabled if all the variables don't have center values.

Reply:
True, but the button _is_ enabled if the variables have center values, custom mode is enabled, but there aren't custom center's. Perhaps I should disable the button under those conditions instead of generating an error?

Reply:
Yeah, let's just disable the button in this case. More consistent that way.